test/e2e/install/install_test.go

```diff
@@ -191,7 +192,7 @@ var _ = Describe("[Install] durability", func() {
 			}
 		})
 
-		It("should be able to delete NUMAResourceOperator CR and redeploy without polluting cluster state", func() {
+		It("[test_id: 47587] should be able to delete NUMAResourceOperator CR and redeploy without polluting cluster state", func() {
```

Comment:
we can write an approxiimation of the upgrade test if we update the NROP deployment -not delete and recreate it.
In addition, the v0.1.0 is way too old. But this is another issue which should be tracked separately.